package.json

```diff
@@ -29,10 +29,8 @@
   },
   "homepage": "https://github.com/Eppo-exp/node-server-sdk#readme",
   "dependencies": {
-    "axios": "^1.6.0",
-    "http-status-codes": "^2.2.0",
-    "lru-cache": "^10.0.1",
-    "@eppo/js-client-sdk-common": "^2.0.0"
+    "@eppo/js-client-sdk-common": "file:../js-client-sdk-common",
```

Comment:
We will publish `js-client-sdk-common` before merging this PR, and replace this with its latest release version.

Reply:
Ah hence the failing lint and type checks